src/PhpSpreadsheet/Worksheet/Worksheet.php

```diff
@@ -3870,4 +3870,36 @@ public function applyStylesFromArray(string $coordinate, array $styleArray): boo
 
         return true;
     }
+
+
+     /**
+      * Copy a row, including style, data and height
+      *
+      * @param int $from row that will be copied
+      * @param int $to row that will receive the copy
+      * @return void
+      */
+      public function copyRow($from, $to) {
+        $this->getRowDimension($to)->setRowHeight($this->getRowDimension($from)->getRowHeight());
```

Comment:
Let's say you want to copy from row 1, which has cells A1/C1/E1, to row 2 which has cells A2/B2/F2. I think your code will replace A2/C2/E2 but leave B2/F2 unchanged, which is probably not what you want - you need to clear B2/F2.